webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -443,24 +551,32 @@ export default class EditChannelSubscription extends PureComponent<Props, State>
 
         const {showConfirmModal} = this.state;
 
-        let confirmDeleteMessage = 'Delete Subscription?';
-        if (this.props.selectedSubscription && this.props.selectedSubscription.name) {
-            confirmDeleteMessage = `Delete Subscription "${this.props.selectedSubscription.name}"?`;
+        let confirmDeleteMessage = '';
+        if (this.props.selectedSubscriptionTemplate) {
+            confirmDeleteMessage = 'Delete Subscription Template?';
```

Comment:
```suggestion
            confirmDeleteMessage = 'Are you sure to delete the subscription template?';
```
I think all such messages should be like this as they are more confirmative